docs/roadmap.md

```diff
@@ -3,15 +3,82 @@
 ## Table of Contents
 - [Status Roadmap](#status-roadmap)
   - [Table of Contents](#table-of-contents)
-  - [2025 H2](#2025-h2)
-    - [2.35](#235)
+  - [2026 H1](#2026-h1)
+    - [2.38](#238)
       - [Features](#features)
-    - [2.36](#236)
+    - [2.39](#239)
       - [Features](#features-1)
-    - [2.37](#237)
+  - [2025 H2](#2025-h2)
+    - [2.35](#235)
       - [Features](#features-2)
-    - [2.38](#238)
+    - [2.36](#236)
       - [Features](#features-3)
+    - [2.37](#237)
+      - [Features](#features-4)
+
+## 2026 H1
+
+### 2.38
+
+Release Epic: https://github.com/status-im/status-desktop/issues/19509
+
+Estimated release: Early February
+
+#### Features
+
+- [Mobile feature parity](https://github.com/status-im/status-desktop/issues/19530)
+  - In Progress ⏳
+  - Includes the features that are missing from out current Mobile app that the one had, like Push notifications, Keycard, etc.
+- [Mobile Dapp Browser](https://github.com/status-im/status-desktop/issues/19535)
+  - In Progress ⏳
+- Polish [Desktop Dapp Browser](https://github.com/status-im/status-desktop/issues/19246)
+  - [FURPS](/docs/FURPS/dapp-browser.md)
+  - In Progress ⏳ 🟩🟩🟩⬜⬜ 68%
+- [Privacy mode](https://github.com/status-im/status-desktop/issues/17619)
+  - [FURPS](/docs/FURPS/privacy-mode.md)
+  - In Progress ⏳ 🟩🟩🟩🟩⬜ 88%
+- [Chat Input Field UX Improvements](https://github.com/status-im/status-desktop/issues/19524)
+- [Unified Adaptive Navigation System](https://github.com/status-im/status-desktop/issues/19458)
+- [Standardized approach for Popups / Menus / Dropdowns](https://github.com/status-im/status-desktop/issues/19493)
+- [Color System Revamp](https://github.com/status-im/status-desktop/issues/19455) - Improved Dark Mode & Updated Palettes
+- [Nimbus Verification Proxy](https://github.com/status-im/status-desktop/issues/19538)
+- [Memory and Performance improvements](https://github.com/status-im/status-desktop/issues/18296)
+  - No provided FURPS at the moment as this is mostly about profiling and fixing issues found.
+  - In Progress ⏳ 🟩🟩🟩⬜⬜ 64%
+- [Improve Token List and Support custom tokens](https://github.com/status-im/status-desktop/issues/19517)
```

Comment:
It might be a good idea to run some testing across the following areas, as they could reveal additional tasks for this epic:
- Permissions
- Mint
- Airdrop
- Profile Showcase / Collectibles
- Join Community, Share Addresses and Community Requirements Check flows
- Manage Token Settings